packages/rest/src/router/routing-table.ts

```diff
@@ -290,7 +290,7 @@ export class Route extends BaseRoute {
     verb: string,
     path: string,
     public readonly spec: OperationObject,
-    protected readonly _handler: Function,
+    protected readonly _handler: Function, // <-- doesn't this Function have a signature?
```

Comment:
The handler function accepts a list of parameters as described in its OpenAPI specification.

See `invokeHandler` below and our docs: https://loopback.io/doc/en/lb4/Routes.html#using-partial-openapi-spec-fragments